cartridge-cli.lua

```diff
@@ -932,14 +937,18 @@ COPY . ${dir}
 
 WORKDIR ${dir}
 
-RUN if [ -f .cartridge.pre ]; then \
-       set -e && . .cartridge.pre && rm .cartridge.pre; \
+RUN if [ -f ${prebuild_script_name} ]; then \
```

Comment:
Please, group this several Runs into one. There are too may layers in result image - it's not ok

Reply:
I'm not sure that this action should be performed in the one RUN.
Are this layers too big?
I need more arguments to understand if it's required.

Reply:
The layer is quite cheap docker primitive, however "best practices" and examples shows us that we should decrease amount of layers. 

https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers

Reply:
In separate PR - ok